### ISSUE TITLE
when showToolbar is set to false, it still flashes briefly

### ISSUE THREAD

**Issue by skyleguy** [stephanrauh/ngx-extended-pdf-viewer#1499]
**Describe the bug**
When opening the pdf viewer for the first time with showToolbar set to false I am seeing the toolbar render initially for a split second and then disappear.

After opening any subsequent time just the black bar of the toolbar shows up for a split second and then disappears. 

I was able to get around it by overriding the intitial toolbar with an empty custom toolbar:

```
<ng-template #hideBlackBar></ng-template>

pdfComponent.customToolbar = this.hideBlackBar;
```

but would like to avoid having to do this. 

**Version info**
- 13.5.2

**Desktop (please complete the following information):**
 - Chrome

If you really can't provide me a reproducer, please tell me how to reproduce the behavior:
1. Set showToolbar to false
2. open a pdf in the viewer
3. See the toolbar flash quickly before disappearing
4. close viewer
5. open again (in the same session)
6. see just the black bar from the toolbar flash quickly before disappearing

**Screenshots**
First Open:
<img width="969" alt="Screen Shot 2022-08-18 at 3 31 05 PM" src="https://user-images.githubusercontent.com/14967057/185478783-1762f543-8d2e-465a-a11b-f9045ce44163.png">

Second Open:
<img width="960" alt="Screen Shot 2022-08-18 at 3 31 14 PM" src="https://user-images.githubusercontent.com/14967057/185478805-24b51e5f-8085-410d-bc36-f6df840bcf81.png">

End Result from either:
<img width="937" alt="Screen Shot 2022-08-18 at 3 31 19 PM" src="https://user-images.githubusercontent.com/14967057/185478832-ad878156-67bc-46d9-ad65-323f8ae7fd18.png">

This is not an issue in the grand scheme of things since there is a workaround but does make the pdf seem a little glitchy when its rendering. Thanks for the awesome library!

**Comment by stephanrauh** [comment]
You've got keen eyes! I couldn't spot the flickering toolbar on my machine, but it sounds plausible. Probably slow networks make the effect more visible and annoying.

I've modified version 15.0.0-alpha.4 to hide the toolbar immediately on `ngOnInit()`. Please run a test and tell me if it helps.

The black bar is the progress indicator (`#loadingBar`). Hide it with a CSS rule if you don't like it. 

BTW, I wonder if the progress bar is broken. I remember it used to be a growing bar, but now it seems to be a full-width black bar until the PDF file has finished rendering.

**Comment by skyleguy** [comment]
looks great in 15.0.0-alpha.4! thank you.

Any issues you see with using 15.0.0-alpha.4 in my Angular 13 app?

**Comment by stephanrauh** [comment]
Feel free to use the alpha version. Unless you're using far-Eastern fonts, there are no known issues. I'm using the extended alpha phases for developing features. In general, that doesn't mean the quality of the library is worse than during the regular-release phase.

It's just that I want to be able to get away with broken versions and half-finished features. But then, this is open-source software published under an Apache 2.0 license, so you don't get any guarantees, anyways. So the only difference between the alpha versions and the first post-alpha version is I'm running a thorough test before calling it a final version.
